.teamcity/Lib/EdFi.LoadTools/buildTypes/BranchBuild.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed to the Ed-Fi Alliance under one or more agreements.
+ * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ * See the LICENSE and NOTICES files in the project root for more information.
+ */
+
+package lib.edFiLoadTools.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+
+object BranchBuild : BuildType ({
+    name = "Branch Build, Package, and Publish"
+    id = RelativeId("EdFiLoadTools_BranchBuild")
+
+    templates(lib.templates.BuildAndPackageSharedLibrary)
+
+    vcs {
+        root(lib.edFiLoadTools.vcsRoots.EdFiStandard, """
```

Comment:
The LoadTest unit test project fails without the Ed-Fi Standard repository.

Reply:
I think if we filter out the tests that it shouldn't be running it would fix that

Reply:
Not sure why we would want to ignore some tests? Isn't it better to go ahead and run these automatically?

Reply:
some of the tests display the mappings between the xml and the json, they don't assert anything. They were setup to help with debugging and have been given a category attribute so that you can filter them out. I'm also fairly sure [this test](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/development/Utilities/DataLoading/EdFi.LoadTools.Test/OAuthTokenHandlerTests.cs#L21) should be failing for you locally and wont work on the build server without some additional changes.

Reply:
Do you know what the command option is to ignore it? And if it would fail locally, it seems strange to keep it around.

Reply:
Found it: ` --filter TestCategory!~"Run Manually"`

Reply:
Successfully ignores those tests when I run manually:

```
dotnet test .\LoadTools.sln --filter TestCategory!~'Run Manually'
```

But TeamCity is writing this as `--filter "TestCategory!~'Run Manually'"` and it isn't working.

![image](https://user-images.githubusercontent.com/9324390/86050971-46ebdc80-ba1a-11ea-8ec3-c8442efd9387.png)

I will have to switch to running the whole command a different way, other than the normal `dotnetTest` step.